src/providers/ldap/sdap_async_nested_groups.c

```diff
@@ -1412,20 +1414,54 @@ sdap_nested_group_single_send(TALLOC_CTX *mem_ctx,
     return req;
 }
 
+static bool must_ignore(struct sdap_nested_group_single_state *state)
+{
+    bool ignore;
+    const char *dn;
+    const char *postfix;
+    struct sdap_search_base **base;
+    int diff;
+
+    DEBUG(SSSDBG_TRACE_INTERNAL, "[TEST] Starting with DN %s\n", state->current_member->dn);
+
+    dn = state->current_member->dn;
+    ignore = false;
+    DEBUG(SSSDBG_TRACE_INTERNAL, "[TEST] ignore_search_bases is %p\n", state->group_ctx->ignore_search_bases);
+    if (state->group_ctx->ignore_search_bases != NULL) {
```

Comment:
Please check this at the very beginning of `must_ignore()` and return immediately without any log message if it is not set.

Reply:
Done